src/charm.py

```diff
@@ -131,12 +130,7 @@ def __init__(self, *args: Any):
 
         self.service_patcher = KubernetesServicePatch(self, [("ldap", GLAUTH_LDAP_PORT)])
 
-        self.loki_consumer = LogProxyConsumer(
-            self,
-            log_files=[str(LOG_FILE)],
```

Comment:
[This](https://github.com/canonical/glauth-k8s-operator/blob/877a05575d1a8caad4bbd0d5da6f272682592f01/src/charm.py#L260) is not needed anymore

Reply:
Good points, thank you! Fixed.